frontend/src/forms/subforms/EvaluationForm.tsx

```diff
@@ -93,14 +93,20 @@ export const validateFinancials = (financials: IFinancial[], nameSpace: string)
   // Yup has major performance issues with the validation of large arrays.
   // As a result, handle the validation manually here.
   let errors = {};
-  financials.forEach((evaluation, index) => {
-    if (evaluation.date && evaluation.key === EvaluationKeys.Appraised && !evaluation.value) {
-      errors = setIn(errors, `${nameSpace}.${index}.value`, 'Required');
-    } else if (
-      evaluation.value &&
-      evaluation.key === EvaluationKeys.Appraised &&
-      !evaluation.date
+  financials.forEach((financial, index) => {
```

Comment:
Yup validation performance is still struggling with large numbers of evaluations, even with the limited scope I was using. Moving this to manual validation significantly improves performance.